core/templates/dev/head/footer_js_libs.html

```diff
@@ -1,5 +1,7 @@
 {% if DEV_MODE %}
   <script src="/third_party/generated/dev/js/third_party.js"></script>
+  <script src="https://npmcdn.com/angular-toastr/dist/angular-toastr.tpls.js"></script>
```

Comment:
Hi @MAKOSCAFEE I added this to manifest.json under `frontend` --
<pre>
"angularToastr": {
  "version": "1.7.0",
  "downloadFormat": "zip",
  "url": "https://github.com/Foxandxss/angular-toastr/releases/download/1.7.0/release.zip",
  "rootDirPrefix": "angular-toastr-",
  "targetDirPrefix": "angular-toastr-",
   "bundle": {
     "css": ["angular-toastr.min.css"],
     "js": ["angular-toastr.tpls.min.js"]
   }
}
</pre>

and then ran `scripts/start.sh` but while fetching the URL it returned a HTTP 404 Error. 
A possible fix for this would be ...?

The library I'm adding - https://github.com/Foxandxss/angular-toastr/